### PR TITLE
SEO-175118 .NET-MAUI-ListView-External-Linking

### PR DESCRIPTION
**Pull request by ghost**
Hi @Girija-Rajendran ,
|Title|Description|
|--|--|
|Task Category| .NET MAUI ListView|
|Content Task Link/Mail Screenshot| NA|
|UX task| NA|
|External link PR link| External link -https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-listview/pull/3
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/175118
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-ListView---Money-Keywords.aspx

 

| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|.net maui listview|10-100 |3FT|To improve our FT experience |added CSS classes alignment format, images aligned in zigzag form, added documentation & Example links, unwanted space removed. |
|.net maui listview example|10-100 |3FT|To move our FT ranking to 1st position | demo link added|
|.net maui listview binding |10-100|3FT|To improve our FT experience | improved on page changes|
|.net maui listview grouping|0-10 |Omit23FT To remove the page from omitted result and improve page experience| added CSS classes alignment format, images aligned in zigzag form, added documentation & Example links, unwanted space removed|
|.net maui listview header|0-10 |Omit - 36FT| To remove the page from omitted result and improve page experience |added corresponding documentation links, arranged images in zigzag form|

 

Worked on 18 keywords in the .NET MAUI ListView and am keeping track of all keyword details in the link below:-
https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-ListView---Money-Keywords.aspx

### CODE REVIEW

**Review by Sharly-Atieno — CHANGES_REQUESTED** [review]
Update PR template

File: README.md [hunk 1]
```diff
@@ -1,14 +1,50 @@
 # Getting Started with the .NET MAUI ListView 
-A quick-start project that helps you to integrate a ListView control in the .NET MAUI application by adding Syncfusion MAUI ListView package from NuGet. This project also explains on how to select items in the list, change its layout, and add a header and footer.
-
-Refer to the following documentation for the Syncfusion .NET MAUI ListView Control: 
-https://help.syncfusion.com/maui/listview/getting-started
-
-Check out this GitHub example for the Syncfusion .NET MAUI ListView Control: 
-https://github.com/syncfusion/maui-demos/tree/master/MAUI/ListView
+The [.NET MAUI ListView control](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) is used to present lists of data in a vertical or horizontal orientation with different layouts virtually.A quick-start project that helps you to integrate a ListView control in the .NET MAUI application by adding Syncfusion MAUI ListView package from NuGet. This project also explains on how to select items in the list, change its layout, and add a header and footer.
```
Comment (Sharly-Atieno) [inline README.md]: Give a single space here:
image

File: README.md [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
+The linear layout arranges items linearly in a single column vertically or a single row horizontally.
+
+### Grid layout
+The grid layout arranges items in a predefined number of columns and the number of columns can be changed.
+
+### Grouping
+A group is a collection of items that belong to a specific category. When grouping is applied, the data is organized into different groups based on key values.
+
+## Related Links
+[Learn More about .NET MAUI ListView](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) <br/>
```
Comment (Sharly-Atieno) [inline README.md]: Give a space in between the links, ......  

Need to be removed in all places.
image

File: README.md [hunk 3]
```diff
@@ -1,14 +1,50 @@
...
+[Download Free Trial](https://www.syncfusion.com/downloads/maui/confirm?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) <br/>
```
Comment (Sharly-Atieno) [inline README.md]: Use the below link for download free trial:
https://www.syncfusion.com/downloads/maui

File: README.md [hunk 4]
```diff
@@ -1,14 +1,50 @@
...
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) <br/>
+[Documentation](https://help.syncfusion.com/maui/listview/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) <br/>
+[View Demos](https://github.com/syncfusion/maui-demos/tree/master/MAUI/ListView?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) <br/>
+[Community Forums](https://www.syncfusion.com/forums/maui?control=sflistview?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) <br/>
```
Comment (Sharly-Atieno) [inline README.md]: (https://www.syncfusion.com/forums/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples)

File: README.md [hunk 5]
```diff
@@ -1,14 +1,50 @@
...
+[Knowledge base](https://support.syncfusion.com/kb?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) <br/>
```
Comment (Sharly-Atieno) [inline README.md]: Use the below link:
https://support.syncfusion.com/kb/cross-platforms/category/76

File: README.md [hunk 6]
```diff
@@ -1,14 +1,50 @@
...
+[Suggest a feature](https://www.syncfusion.com/feedback/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples)
```
Comment (Sharly-Atieno) [inline README.md]: [Suggest a feature or report a bug]

File: README.md [hunk 7]
```diff
@@ -1,14 +1,50 @@
...
+Syncfusion's [.NET MAUI controls](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) library is the only suite that you will ever need to build an application since it contains over 80 high-performance, lightweight, modular, and responsive UI controls in a single package. In addition to ListView, we provide popular .NET MAUI Controls such as [DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples), [Charts](https://www.syncfusion.com/maui-controls/maui-cartesian-charts?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples), [Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples), [ListView](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples), and [Excel](https://www.syncfusion.com/document-processing/excel-framework/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples).
```
Comment (Sharly-Atieno) [inline README.md]: Since you are working on ListView control, you need to remove the below content
image

File: README.md [hunk 8]
```diff
@@ -1,14 +1,50 @@
...
+
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+ 
+<p align="center">
+  <a href="mailto:sales@syncfusion.com?Subject=Syncfusion .NET MAUI ListView - Visual Studio Marketplace">sales@syncfusion.com</a> | Toll Free: 1-888-9 DOTNET
+</p>
```
Comment (Sharly-Atieno) [inline README.md:57]: Update the correct style here:

**Review by Sharly-Atieno — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Added space.

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Removed <br/> in all places

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md]: Added correct link

File: README.md (hunk 4 above)
Comment (ghost) [reply README.md]: added correct link.

File: README.md (hunk 5 above)
Comment (ghost) [reply README.md]: Added correct link.

File: README.md (hunk 6 above)
Comment (ghost) [reply README.md]: Resolved

File: README.md (hunk 7 above)
Comment (ghost) [reply README.md]: Removed ListView

File: README.md (hunk 8 above)
Comment (ghost) [reply README.md:57]: Updated correct style.

**Review by Sharly-Atieno — APPROVED** [review]

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**